### ISSUE TITLE
Support URL Protocol

### ISSUE THREAD

**Issue by LinWanCen** [kingToolbox/WindTerm#102]
Hope to support the browser to open the software.
It is convenient to create a web page that generates a test environment link list from the IP port user password, and click it to open it directly with WindTerm:
```html
<a href="winterm:ssh?host=1.2.3.4&user=root&pass=root">Open with WindTerm</a>
```

Can write URL Protocol and open\command to win Registry like this:
```
Windows Registry Editor Version 5.00

[HKEY_CLASSES_ROOT\mobaxterm]
@="URL:winterm Protocol Handler"
"URL Protocol"=""

[HKEY_CLASSES_ROOT\winterm\DefaultIcon]
@="D:\\Program Files (x86)\\WindTerm\\WindTerm_Win10.exe"

[HKEY_CLASSES_ROOT\winterm\shell]

[HKEY_CLASSES_ROOT\winterm\shell\open]

[HKEY_CLASSES_ROOT\winterm\shell\open\command]
@="D:\\Program Files (x86)\\WindTerm\\WindTerm_Win10.exe \"%1\""
```

**Comment by kingToolbox** [comment]
Thank you for such a good suggestion. I think this job is more suitable for the installer. There is no installation program yet. I will consider adding this feature when I make it.

**Comment by LinWanCen** [comment]
Is it possible to write these only during installation? Hope to keep it portable and provide a repair URL Protocol menu option.

**Comment by kingToolbox** [comment]
Yes, it is possible. In addition to the installation script, we can provide a separate button in WindTerm to re-register the information. This will ensure that WindTerm is still portable.
